std/php7/Global.hx

```diff
@@ -82,4 +83,133 @@ extern class Global {
 	**/
 	@:overload(function(original:String,alias:String):Bool {})
 	static function class_alias( original:String, alias:String, autoload:Bool ) : Bool ;
+
+	/**
+		@see http://php.net/manual/en/function.count.php
```

Comment:
I think there is 1 tab but they are two now (in the functions I added at least).